.claude/commands/design-to-dev.md

```
@@ -0,0 +1,157 @@
+# /design-to-dev — Figma to Component Pipeline
+
+Converts a Figma URL into committed component code + Storybook stories + visual comparison report.
+
+## Input
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)